programs/sponsors/sponsors.md

```diff
@@ -0,0 +1,61 @@
+# Sponsors
+
+JSON Schema uses [OpenCollective](https://opencollective.com/json-schema) and [GitHub Sponsors](https://github.com/sponsors) to gather money. This money is used to cover project expenses in a transparent way ([See the OpenCollective documentation](https://opencollective.com/how-it-works)), and your donation will help to sustain and grow the Project.
+
+Please go to our [OpenCollective page](https://opencollective.com/json-schema) or [GitHub Sponsors page](https://github.com/sponsors/json-schema-org) to become a backer or a sponsor.
+
+## Benefits of being a backer
+
+Backers donate $5 per month to the project, and get the following benefits:
+
+- Visibility on the GitHub main project page in the "sponsors" section.
+
+## Benefits of being a bronze sponsor
+
+Bronze sponsors donate $100 per month to the project, and get the following benefits:
+
+- Visibility on the front page of [https://json-schema.org](https://json-schema.org) in the "sponsors" section (about 110,000 views/month on November, 2023).
+- Visibility on the GitHub main project page in the "sponsors" section.
+- "Thank you" tweet from [@jsonschema](https://twitter.com/jsonschema).
+- "Thank you" post in Linkedin from [@jsonschema](https://www.linkedin.com/company/jsonschema).
+
+## Benefits of being a silver sponsor
+
+There is a limit to 10 silver sponsors.
+
+Silver sponsors donate $500 per month to the project, and get the following benefits:
+
+- Same benefits as bronze sponsors.
+- Visibility on the Newsletter footer.
```

Comment:
We mean "a mention" here rather than "Visibility" I think right? As in we will mention/shout out the sponsor in these 2 places?

Reply:
Fixed in the last push.